README.md

```diff
@@ -1,89 +1,74 @@
-# Getting Started ![version](https://img.shields.io/npm/v/react-native-plaid-link-sdk)
+# Plaid React Native SDK
 
-In your react-native project directory:
+![version](https://img.shields.io/npm/v/react-native-plaid-link-sdk)
+[![License](https://img.shields.io/github/license/plaid/react-native-plaid-link-sdk)](https://github.com/plaid/react-native-plaid-link-sdk/blob/master/LICENSE)
 
-```sh
-npm install --save react-native-plaid-link-sdk
-```
+The Plaid React Native SDK provides the client-side component that your users will interact with in order to link their accounts to Plaid and all you to access their accounts via the Plaid API. 
 
-For a sample app demonstrating a minimal integration with this SDK, see the [Tiny Quickstart (React Native)](https://github.com/plaid/tiny-quickstart/tree/main/react_native). For a full guide and migration guides please vist our [docs][plaid_rndocs].
+For more information about Plaid Link check out our
+[introduction documentation](https://plaid.com/docs/link/#introduction-to-link).
```

Comment:
I strongly think we should retain the link to the tiny quickstart, we've found it is very useful for a lot of developers to have a working end to end example that incorporates both the frontend and the backend in a single repo, especially if they're mobile-first (because Plaid didn't add mobile SDKs until like 2019, a lot of our docs for mobile have historically assumed that you were adding mobile support to an already working Plaid backend, but this is less and less frequently the case). You're welcome to also link to the Example directory as well!

Reply:
Good to know. I added the link back in. LMK if that repo is something else I should consider updating :)